drivers/entropy/entropy_sam.c

```diff
@@ -56,17 +56,6 @@ static int entropy_sam_wait_ready(Trng * const trng, uint32_t flags)
 		if (timeout-- == 0) {
 			return -ETIMEDOUT;
 		}
-
-		if ((flags & ENTROPY_BUSYWAIT) == 0U) {
```

Comment:
It seems flags is not necessary anymore so I was wondering if it is not possible to remove it.

Reply:
I could be used in future, but that is just an internal function. Will remove it and if needed we re-introduce it later.